.github/dependabot.yml

```diff
@@ -3,4 +3,4 @@ updates:
   - package-ecosystem: "gradle"
     directory: "/"
     schedule:
-      interval: "daily"
+      interval: "monthly"
```

Comment:
Haha ... too noisy? :D may be weekly?

Reply:
Nah, we decided in the team that monthly is perfectly fine for passive dependency bump :)